src/client/app/components/LoginComponent.tsx

```diff
@@ -5,24 +5,27 @@
 import * as React from 'react';
 import { useRef, useState } from 'react';
 import { FormattedMessage } from 'react-intl';
-import { useNavigate } from 'react-router-dom';
 import { Button, Form, FormGroup, Input, Label } from 'reactstrap';
 import { authApi } from '../redux/api/authApi';
 import { showErrorNotification, showSuccessNotification } from '../utils/notifications';
 import translate from '../utils/translate';
 
 
+interface LoginProp {
+	handleClose: () => void;
+}
+
 /**
+ * @param handleClose Function to close modal after login
```

Comment:
I know we talked about this once before but I'm still getting an ESLint warning in VSC for not having the parameter handleClose.handleClose. Could it be added to remove this? If you don't see it or want me to do it then let me know. I do see it passes all the checks on GitHub but it would still be nice to clean this up.

Reply:
I accidentally pushed to a different branch. It should be corrected now.